rockr/views.py

```diff
@@ -93,7 +94,63 @@ def user_goals(user_id):
         ).first()
         db_manager.delete(ug)
         return format_response(204, None)
-
+    
+#Added for the User Band queries
+@app.route('/user_band/<int:user_id>', methods=["GET","POST","DELETE"])
+def user_band(user_id):
+    if request.method == "GET":
+        umi = UserBand.query.filter_by(user_id=user_id)
+        bandsids = [Band.query.get(b.band_id) for b in umi]
+        return format_response(200, serialize_query_result(bandsids))
+    #For a new User Band Adding Query
+    elif request.method == "POST":
+        ub = UserBand.query(func.max(UserBand.id))
+        highestid = ub.id;
+        highestid = highestid+1;
+        ubnew = {
+            "id": highestid,
+            "user_id": user_id,
+            "band_id": int(request.args["id"])
+        }
+        db_manager.insert(ubnew)
+        return format_response(201, None)
+    else:
+        ub = UserBand.query.filter_by(
+            user_id=user_id,
+            band_id=int(request.args["id"])
+        ).first()
+        db_manager.delete(ub)
+        return format_response(204, None)
+    
+#In case you want to modify the functions to be like this
+@app.route('/get_user_band/<int:user_id>', methods=["GET"])
+def get_user_band(user_id):
```

Comment:
this endpoint is redundant. This behavior should be covered by hitting `user_band(user_id)` with a GET request.

Reply:
I thought it might be needed for in case it is triggered by a button or something.

Reply:
I don't think you understand what I'm saying. We can call this same endpoint from the declaration for `user_band`. We should get rid of `get_user_band`, `add_user_band`, and `remove_user_band`

Reply:
Well what if they need to do it through a button, instead of automatically? Will it hurt to keep it? Like how you added get_user_role and get_role. I just don't want the button to trigger it the wrong way if possible.

Reply:
Okay I looked over it more and I removed them. I pushed the newest version and maybe now it should be better.